src/renderer/scss-partials/_ft-list-item.scss

```diff
@@ -2,6 +2,36 @@
 $thumbnail-overlay-opacity: 0.85;
 $watched-transition-duration: 0.5s;
 
+@mixin is-side-nav-open {
+  @at-root {
+    .isSideNavOpen &,
+    .isSideNavOpen#{&} {
+      @content;
+    }
+  }
+}
+
+@mixin fixed-top-bar {
+  position: fixed;
+  inset-block-start: 60px;
+  inset-inline-end: 0;
+  inline-size: calc(100% - 80px);
+  margin-inline: 0;
+  z-index: 3;
+
+  @include is-side-nav-open {
+    inline-size: calc(100% - 200px);
+  }
+
+  @media only screen and (width <= 680px) {
+    inline-size: 100%;
+
+    @include is-side-nav-open {
+      inline-size: 100%;
+    }
+  }
+}
+
```

Comment:
As far as I can tell this is only used in the `Playlist.scss` and `ft-refresh-widget.scss` files, so it probably makes more sense to place it in a new partial file, as it seems unrelated to the rest of the stuff in this file.